src/Css_AtomicTypes.re

```diff
@@ -1535,3 +1535,77 @@ module FontDisplay = {
     | `fallback => "fallback"
     | `optional => "optional";
 };
+
+module Counter = {
+  type style = [ ListStyleType.t | `unset];
+  type t = [ | `counter(string, style) | `counters(string, string, style)];
+
+  let counter = (~style=`unset, name) => `counter((name, style));
```

Comment:
I'm not sure about those two names, they look a bit odd on a call site `Counter.counter(...)`. I don't have a better idea, though. `single` & `many`? `make` & `makeMany`? It doesn't seem consistent with other modules. Or maybe it's not important if I open this module in the main `Css.re` file?

Reply:
maybe just have a `type t = (string, string, style)`, because counters have only one form. If style is unset, your implementation is already equivalent to `(string, string)`.

You can only keep one constructor with default values `let counters = (~style=`unset, ~separator="", name) =>`

That should be enough

Reply:
One thing is that I think that type style should use a `CounterStyle.t`, not a `ListStyleType.t`

Reply:
What you recommend is creating a separate type, equal to `ListStyleType.t`  implementation-wise?

Reply:
`counter` and `counters` are two different values in CSS:
 * https://developer.mozilla.org/en-US/docs/Web/CSS/counter
 * https://developer.mozilla.org/en-US/docs/Web/CSS/counters
Do you think there should be two separate modules?

Reply:
There is one issue with this solution, though. If I define `t` like `(string, string, style)`, I cannot merge the type in `rei` file. It throws an error that `Types.Counter.t` isn't a polymorphic variant type. I'm not ReasonML expert, so I don't have any idea how to work around that.

```reasonml
let contentRule:
  [<
    Types.Content.t
    | Types.Counter.t // NOPE!
    | Types.Counters.t
    ...
  ] => rule;
```

Reply:
> What you recommend is creating a separate type, equal to `ListStyleType.t` implementation-wise?

yep

Reply:
> There is one issue with this solution, though. If I define `t` like `(string, string, style)`, I cannot merge the type in `rei` file. It throws an error that `Types.Counter.t` isn't a polymorphic variant type. 

Exact, forgot about that point. 
We might do something like Url: `module Counter = { type t = [ | 'counter(string, string, style) ]; ... }`